src/capybara.js

```diff
@@ -269,8 +269,20 @@ Capybara = {
     return true;
   },
 
+  sendKeys: function (index, keys) {
+    var strindex, length;
+
+    length = keys.length;
+
+    this.focus(index);
```

Comment:
OK, I see now. The current implementation will still trigger focus twice in `set`. So perhaps the best option would be to duplicate https://github.com/maxnordlund/capybara-webkit/blob/add-send_keys/src/capybara.js#L281-L283 in place of https://github.com/maxnordlund/capybara-webkit/blob/add-send_keys/src/capybara.js#L306 (as was before). Or, extract out another helper method.